nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/document/AddPermissionTest.java

```diff
@@ -21,6 +21,9 @@
 
 import static java.util.Collections.singletonList;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.arrayWithSize;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
```

Comment:
We usually rely on the `org.junit.Assert` package but I guess we don't really have any conventions about it.

Reply:
yep,

Reply:
We probably need one... I would prefer to stick to `org.junit.Assert` also.
WDYT @efge?

Reply:
I also prefer `org.junit.Assert` if it's enough. That's been our convention in the code for quite a while.
Note that doing a static import of `instanceOf` is especially bad because it's very close to the keyword `instanceof`, so may be confusing.

Reply:
yep,